aea/crypto/helpers.py

```diff
@@ -27,7 +27,10 @@
 
 from fetchai.ledger.crypto import Entity  # type: ignore
 
+from web3 import Web3
```

Comment:
Given that this depends on web3, I am tempted to move this helper function into the ethereum crypto module as a standalone function. Also since we expect it to be used in skills drop the `_` from the name and align the naming to the `tx_nonce`